packages/web/src/scss/components/Tooltip/index.html

```diff
@@ -241,19 +315,28 @@ <h2 class="docs-Heading">Tooltip on Click (JavaScript)</h2>
         type="button"
         id="tooltip-trigger"
         class="Button Button--primary Button--medium"
-        data-spirit-target="#my-js-controlled-tooltip"
         data-spirit-toggle="tooltip"
-      >
+        data-spirit-target="#my-js-controlled-tooltip"
+        >
         Toggle tooltip
       </button>
-
-      <div class="TooltipWrapper d-inline-block">
+    
+      <div class="TooltipWrapper d-inline-block" data-spirit-element="tooltip-wrapper">
```

Comment:
This is one of the original examples, is the addition of the `data` attribute here (and also below) OK?

Reply:
added more demos to cover all examples (with and without floating ui)